neo/rawio/intanrawio.py

```diff
@@ -335,12 +343,21 @@ def _get_signal_size(self, block_index, seg_index, stream_index):
             mask = self.header["signal_channels"]["stream_id"] == stream_id
             signal_channels = self.header["signal_channels"][mask]
             channel_ids = signal_channels["id"]
-            channel_id_0 = channel_ids[0]
-            size = self._raw_data[channel_id_0].size
+
+            stream_name = self.header["signal_streams"][stream_index]["name"][:]
```

Comment:
Remind me the `[:]` I always forget this :)

Reply:
You mean the why? One is a reference the other gets the data which I need here to have the string version.

Reply:
Done.